README.md

```diff
@@ -65,7 +65,7 @@
   - always prefer functional programing over procedual. Meaning write a lot of small functions
     instead of one big function that does everything. Doing so will facilitate function chains, which in turn improve code readability
 
-  - a `class` should have at the most 100 lines of code.
+  - large line count for a `class` is usually a sign it can be split in smaller components. Look for this sign during code reviews.
```

Comment:
Though I totally understand the appeal of the rule, I usually suggest this is watched for during code reviews, though, since enforcing this could promote the use of functions with side-effects or the lets-do-everything-in-one-line mentality to reduce the number of lines in the code,